src/taskStatusManager.js

```diff
@@ -0,0 +1,15 @@
+import toDoList from './todolist.js';
+
+export default function updateListItem(status, index) {
+  toDoList.updateStatus(status, index);
+}
+
+document
+  .getElementById('clear-completed')
+  .addEventListener('click', () => {
+    const elements = document.getElementsByClassName('task-status');
```

Comment:
I think using queryselector or queryselectorAll as the case may be, would make your code better as get element by classname tends to select the first element that has that class